dcpy/models/product/dataset/metadata.py

```diff
@@ -142,16 +147,51 @@ def get_columns(self, metadata: Metadata) -> list[Column]:
         return cols
 
 
+class ZipFile(File, extra="forbid"):
+    type: Literal["Zip"] = "Zip"
+    contains: list[File]
+
+
 class Package(BaseModel, extra="forbid"):
+    # TODO: should these just be combined into one flat list of files? Why have three distinctions
```

Comment:
specifically, why have these three different attributes on the class, rather than just files, e.g. `List[File]`